hack/label_nodes/label_nodes.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python3
```

Comment:
some comment on how this script can be used would be good and the args they need to pass

Reply:
added. let me know if it is too detailed, or if perhaps i should include a concrete example use case?